app/build.gradle

```
@@ -21,7 +24,7 @@ dependencies {
     implementation "org.apache.servicemix.bundles:org.apache.servicemix.bundles.kafka-clients:$kafkaClientVersion"
 
     implementation "org.slf4j:slf4j-api:$slf4jVersion"
-    implementation "org.pf4j:pf4j:${pf4jVersion}"
+    implementation ("org.pf4j:pf4j:${pf4jVersion}")
```

We can probably revert this change:
```suggestion
    implementation "org.pf4j:pf4j:${pf4jVersion}"
```